sdk/src/keystore/backend/mem.rs

```diff
@@ -0,0 +1,274 @@
+//! In Memory Keystore Backend that supports different cryptographic key operations such as key generation, signing, and public key retrieval.
```

Comment:
Not pressing, but `In Memory` generally uses a hyphen to avoid confusion when reading: `In-Memory`